api/middleware.go

```diff
@@ -104,6 +107,23 @@ func MetricsMiddleware(m metrics.RequestMetrics, logger log.Logger) func(next ht
 	}
 }
 
+// Bin request durations to make it easier to search
+// for slow queries in Nexus logs.
```

Comment:
FWIW, GCP Logs Explorer does allow arithmetic comparisons, see [example](https://cloudlogging.app.goo.gl/iJQ585LPytqCJW1u9). And of course we slice and dice in whatever way we want if we download the logs to our local machine. So I don't see the benefit of this field, but it also doesn't hurt me :)

Reply:
Agreed, it's also a fallback in case gcp keeps eating the `time`/`latency` field for whatever reason.